ocpp-v2/src/main/java/eu/chargetime/ocpp/v201/model/messages/PublishFirmwareStatusNotificationRequest.java

```diff
@@ -0,0 +1,262 @@
+   * Gets custom data
+   *
+   * @return Custom data
+   */
+  @Nullable
+  public CustomData getCustomData() {
+    return customData;
+  }
+
+  /**
+   * Sets custom data
+   *
+   * @param customData Custom data
+   */
+  public void setCustomData(@Nullable CustomData customData) {
+    if (!isValidCustomData(customData)) {
+      throw new PropertyConstraintException(customData, "customData is invalid");
+    }
+    this.customData = customData;
+  }
+
+  /**
+   * Returns whether the given customData is valid
+   *
+   * @param customData the customData to check the validity of
+   * @return {@code true} if customData is valid, {@code false} if not
+   */
+  private boolean isValidCustomData(@Nullable CustomData customData) {
+    return customData == null || customData.validate();
+  }
+
+  /**
+   * Adds custom data
+   *
+   * @param customData Custom data
+   * @return this
+   */
+  public PublishFirmwareStatusNotificationRequest withCustomData(@Nullable CustomData customData) {
+    setCustomData(customData);
+    return this;
+  }
+
+  /**
+   * Gets the progress status of the publishfirmware installation.
+   *
+   * @return The progress status of the publishfirmware installation
+   */
+  public PublishFirmwareStatusEnum getStatus() {
+    return status;
+  }
+
+  /**
+   * Sets the progress status of the publishfirmware installation.
+   *
+   * @param status The progress status of the publishfirmware installation
+   */
+  public void setStatus(PublishFirmwareStatusEnum status) {
+    if (!isValidStatus(status)) {
+      throw new PropertyConstraintException(status, "status is invalid");
+    }
+    this.status = status;
+  }
+
+  /**
+   * Returns whether the given status is valid
+   *
+   * @param status the status to check the validity of
+   * @return {@code true} if status is valid, {@code false} if not
+   */
+  private boolean isValidStatus(PublishFirmwareStatusEnum status) {
+    return status != null;
+  }
+
+  /**
+   * Gets required if status is Published. Can be multiple URI’s, if the Local Controller supports
+   * e.g. HTTP, HTTPS, and FTP.
+   *
+   * @return Required if status is Published
+   */
+  @Nullable
+  public String[] getLocation() {
+    return location;
+  }
+
+  /**
+   * Sets required if status is Published. Can be multiple URI’s, if the Local Controller supports
+   * e.g. HTTP, HTTPS, and FTP.
+   *
+   * @param location Required if status is Published
+   */
+  public void setLocation(@Nullable String[] location) {
+    if (!isValidLocation(location)) {
+      throw new PropertyConstraintException(location, "location is invalid");
+    }
+    this.location = location;
+  }
+
+  /**
+   * Returns whether the given location is valid
+   *
+   * @param location the location to check the validity of
+   * @return {@code true} if location is valid, {@code false} if not
+   */
+  private boolean isValidLocation(@Nullable String[] location) {
+    return location == null || (location.length >= 1);
```

Comment:
Oops, indeed, I missed this.

Is it ok if I force-push to fix this, rather than stacking a new commit on top? I could also rebase it to the current master while at it.

Reply:
You should be able to just push a new commit to the PR and github should be able to detect that in a good way.

Reply:
Done. This fix is now a commit past the version bump, though.